doc/design/OrionMongoSink.md

```diff
@@ -54,23 +54,28 @@ Assuming `mongo_username=myuser` as configuration parameter, then `OrionMongoSin
     > show databases
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/0119928dc774062e17071dc7823944f3cc04fb19